### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by jakub-bartoszek**

### CODE REVIEW

File: src/components/common/SmallTile/SmallTile.jsx [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
+     <Genres genreIds={movie.genre_ids} />
+    </MovieInfo>
+    {movie.vote_count > 0 ? (
+     <Rating>
+      <IconStar />
+      <Rate>{movie.vote_average.toFixed(1)}</Rate>
+      <Votes>{`${movie.vote_count} votes`}</Votes>
+     </Rating>
+    ) : (
+     <Votes>No votes</Votes>
```
Comment (AndriiStafiniak) [inline src/components/common/SmallTile/SmallTile.jsx]: suggest adding "yet" at this point as in the project
Zrzut ekranu 2023-12-1 o 13 00 52

**Comment by AndriiStafiniak** [comment]
I also suggest improving the styles on smaller screens
reduce the font to 13px and increase the spacing
Zrzut ekranu 2023-12-1 o 13 04 09

**Comment by AndriiStafiniak** [comment]
error information is not centered. we need margin top 180 px.
Zrzut ekranu 2023-12-1 o 13 12 57

**Review by AndriiStafiniak — APPROVED** [review]
very nice changes, only a few small details related to styles need to be improved. and we will be able to merge

**Comment by jakub-bartoszek** [comment]
I fixed that

**Pull request merged**